Formula/node.rb

```diff
@@ -34,6 +34,7 @@ class Node < Formula
     depends_on "gcc"
   end
 
+  fails_with :clang if DevelopmentTools.clang_build_version < 1100
```

Comment:
```suggestion
  fails_with :clang do
    build 1099
    cause "Node requires Xcode CLT 11+"
  end
```
`fails_with` already understands build versions, so there's no need to gate it with `if`.